### DISCUSSION TITLE
Como ingressar no mercado de trabalho?

### DISCUSSION THREAD

**Discussion by camila1467** [backend-br/forum#107] [Q&A] [answered]
Pessoal, estou fazendo ciência da computação, indo pro quinto semestre, porém, me vejo em uma situação que parece que não sei nada para conseguir entrar no mercado, eu estou querendo me especializar em Java, mas ainda estou travada em estrutura de dados (c que aprendo na faculdade). As vezes bate um desespero pois todos os currículos que mando de estágio não sou chamada nem pra entrevista. Gostaria de um norte, em pessoas que começaram assim e o que fizeram, eu sei Java e poo porém não muito, sei um pouco sql e sei C. Falando em c, é mais fácil aprender estrutura em Java ou C? Desde já, obrigada por lerem.

**Comment by Roneycacau** [comment, marked as answer]
Olha eu não sou muito bom nisso de programar ainda e tive a sorte de conseguir um estágio numa empresa muito boa. chama Iteris, segue eles no linkedin https://www.linkedin.com/company/iteris-consultoria-e-software/?originalSubdomain=br  que umas duas vezes por ano eles abrem inscrição pro programa de estágio deles. Foi minha única esperiência profissional em programação e mesmo só com isso já bateu um monte de propostas de entrevista.
No mais tenta dar uma ajeitada no linkedinho, se cadastra numas vagas por lá. Sai disparando candidatura por aí. Me cadastrei na Revelo e parece promissor.  E, se é o que realmente quer fazer, não desistir. É frustrante não conseguir nada mas depois da primeira experiência o restante vai acabar aparecendo.

Quanto a estrutura de dados, em Java meio que já existem as estruturas então você não precisa criar elas como em C. E como não tem ponteiro fica mais fácil pra fazer as paradinhas em Java. Mas aprender em C vai te dar uma base melhor de como essas estruturas funcionam. É doído mas de ta mais base. E se quiser dar uma olhada tem essa playlist da Loiane que ensina um pouco sobre estrutura de dados com Java: https://www.youtube.com/playlist?list=PLGxZ4Rq3BOBrgumpzz-l8kFMw2DLERdxi

**Reply by jeancsanchez** [reply]
Muito bom esses pontos levantados. Mais dois pontos que eu adicionaria seria:
- Inglês
- Faculdade

O segundo ponto é meio polêmico, mas pra mim, me ajudou muito no começo no quesito conhecer e estar em contato com pessoas que já atuavam na área. Não acho que seja um impeditivo para ingressar na área, mas na minha humilde opinião, lá você fica muito exposto a oportunidades.
